updater/cmd/updater/main.go

```diff
@@ -0,0 +1,35 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/observiq/observiq-otel-collector/internal/version"
```

Comment:
Do it make sense to break this out into its own package? Seems weird to share a internal package from another module.

Reply:
Honestly, not really sure.

It does feel weird, but OTel does it all the time, and it seems to work fine for them. 

I don't really have any major objections to breaking it out, but it also feels weird to have this as a public package when it requires specific build steps and isn't really meant to be used outside this repo.

Reply:
@StefanKurek Do you also think this should be broken into a separate package? Or possibly duped into a package internal to the new module?

IMO the way it is now is the most straightforward, but I can see how it's a little wonky.

Reply:
@BinaryFissionGames @cpheps honestly i'll go by what corbin says as he's the go master

Reply:
it is wonky, I think the most correct go way would be to copy the version package into an internal package in the updater. Right now the updater is importing the collector as a dependency for that one package even though it doesn't actually need the collector.